corehq/apps/smsbillables/models.py

```diff
@@ -255,6 +259,7 @@ class SmsBillable(models.Model):
     the monthly Invoice.
     """
     gateway_fee = models.ForeignKey(SmsGatewayFee, null=True, on_delete=models.PROTECT)
+    twilio_gateway_fee = models.DecimalField(null=True, max_digits=10, decimal_places=4)
```

Comment:
we might want to rename this field to be something more general like `direct_gateway_fee`, just in case in the future we have another gateway that we can do direct lookups with